NetCoreDemos/DevExtreme.NETCore.Demos.csproj

```diff
@@ -24,7 +24,7 @@
     <PackageReference Include="Microsoft.Azure.Storage.Blob" Version="11.1.0" />
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.1.11" />
     <PackageReference Include="DevExtreme.AspNet.Data" Version="2.6.1" />
-    <PackageReference Include="DevExtreme.AspNet.Core" Version="20.2.0" />
+    <PackageReference Include="DevExtreme.AspNet.Core" Version="20.2.2" />
```

Comment:
why?

Reply:
It was modified automatically when all preparation scripts were executed.

Reply:
I rolled back this change.